addons/eventbus/README.md

```diff
@@ -36,6 +36,10 @@ creating a singleton instance of ```ApplicationContextEventBroker```:
  }
 ```    
 
+# Event Bus Topics
+Events can be published with a string based topic. This will distinguish which listener methods will be called.
+Therefore a topic must be provided when publishing events and at the listener methods.
```

Comment:
The topic can be left out as well, in which case it will default to an empty string, right?